### DISCUSSION TITLE
Show block toolbar on top of centered blocks

### DISCUSSION THREAD

**Discussion by davidnb3** [WordPress/gutenberg#46739] [General]
Hello,

Would it be possible to display the block toolbar just above the text inside a centered block, or in general above the text in every block ?

Centered text blocks that aren't inside a group or column block have a full page width which leads to the toolbar being shown at the edge of the page (as you can see on the screenshots).

![Bildschirmfoto 2022-12-22 um 12 15 49 PM](https://user-images.githubusercontent.com/56735699/209125120-a18cecc6-a27e-4623-a1cd-d287ab5472a7.png)
![Bildschirmfoto 2022-12-22 um 12 05 11 PM](https://user-images.githubusercontent.com/56735699/209125123-eb34958e-dcac-4a8a-9f8e-173ffb60258c.png)

Would appreciate some feedback on that :) Thanks!

**Comment by devfle** [comment]
This is really a problem. I think the distances for the user to make changes should be as short as possible. Also, this way it is not certain to which block the toolbar belongs.
